melange.yaml

```diff
@@ -58,6 +58,22 @@ subpackages:
             [ -f /usr/bin/ldd-check ]
             [ -x /usr/bin/ldd-check ]
 
+  - name: gem-check
+    options:
+      no-provides: true
+    dependencies:
+      runtime:
+        - apk-tools
+        - busybox
```

Comment:
it clearly _does_ require ruby. so lets at least call that out maybe? i understand you're wanting to use the ruby that is installed with the gem.